src/components/weatherCard/weatherCard.js

```diff
@@ -1,12 +1,17 @@
 import React from "react";
 import { getPosition } from "../../helpers/getPosition";
 import { useDispatch, useSelector } from "react-redux";
+import { Spinner, WeatherCardWrapper } from "./styles";
 
 export const WeatherCard = () => {
   const weatherDispatch = useDispatch();
   const weather = useSelector((state) => state);
 
   const getWeather = async () => {
+    weatherDispatch({
+      type: "SET_LOADING_TRUE",
+      weatherRecieved: false,
```

Comment:
```suggestion
      payload: false,
```

но как минимум это юзлес, можеш прям в редюсере это сетать